viewflow/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
I removed it because this don't have anything with tools namespace actually declared in xml.
This only has android namespace declared. 
Thus,
```
 xmlns:tools="http://schemas.android.com/tools" 
```
is not used so we can safely remove it.